.github/workflows/release.yml

```diff
@@ -2,6 +2,15 @@ name: Upload Python package
 on:
   release:
     types: [created]
+  pull_request:
+    paths:
+      - .github/workflows/release.yml
+
+# When this workflow is queued, automatically cancel any previous running
+# or pending jobs from the same branch
+concurrency:
+  group: release-${{ github.head_ref }}
+  cancel-in-progress: true
```

Comment:
cc @ayushdg okay with leaving this in for other PRs modifying the workflow?

Reply:
Yeah. I'd actually prefer this so that we start testing CIBW updates in an automated way as well.